config/horizon.json

```diff
@@ -34,6 +34,7 @@
 		"current-site/domain-warning": true,
 		"current-site/notice": true,
 		"current-site/stale-cart-notice": false,
+		"design-picker/goal-centric": true,
```

Comment:
Let's keep the feature flag disabled for Horizon for now since we're still early in development.